### PR TITLE
AC-755 Fixed LocationRoomDAO tests

### PR DESCRIPTION
**Pull request by rishabh-997**
## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-755

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/dao/LocationRoomDAO.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
 public interface LocationRoomDAO {
 
     @Insert
-    Single<Long> saveLocation(LocationEntity entity);
+    void saveLocation(LocationEntity entity);
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/dao/LocationRoomDAO.java:29]: Shouldn't this method return the new inserted location id? What is the change here?

Comment (rishabh-997) [reply openmrs-client/src/main/java/org/openmrs/mobile/dao/LocationRoomDAO.java:29]: Using Single<Long> wasn't adding files to the database so I changed it to void and worked fine. I thought of writing other tests and look into it later but it completely slipped off my mind.

Comment (rishabh-997) [reply openmrs-client/src/main/java/org/openmrs/mobile/dao/LocationRoomDAO.java]: anyways, fixed it... I have made return type as `Long` which is the same as other `entities`

**Comment by VibhorChinda** [comment]
@rishabh-997 just squash commits. Everything else looks fine to me :)

**Pull request merged**